server.js

```diff
@@ -1,16 +1,38 @@
-const express = require('express');
+const express = require("express");
 const app = express();
+const morgan = require("morgan");
+const cors = require("cors");
 
-app.get('/getme', (req, res) => {
-  res.json({})
+app.use(cors());
+app.use(morgan("dev"));
+app.use(express.json());
+
+const bodyLogger = (req, res, next) => {
+  console.log(req.body);
+  next();
+};
+const fullName = (req, res, next) => {
+  const { firstname, lastname } = req.body;
+  if (firstname && lastname) {
+    res.locals.fullname = firstname + lastname;
```

Comment:
Just a minor: The fullname will be nicer to read, if you put a blank in between, e.g.
firstname + ' ' + lastname

Reply:
I changed that bit. Thanks! :)